doc/holochain_101/src/new_project.md

```diff
@@ -0,0 +1,21 @@
+# Create A New Project
+
+The command line tools discussed in the last article can be used to easily create a new folder on your computer, that contains all the initial folders and files needed for a Holochain application. 
+
+You will typically want to start a new Holochain application this way, since creating them all by hand would waste your time.
+
+In your terminal, change directories to one where you wish to initialize a new Holochain app. The command will create a new folder within the current directory for your app.
+
+Come up with a name for your application, or at least for your project folder.
+
+Copy or type everything after the `$` sign below into your terminal, except replace `your_app_name` with the name you came up with. Press `Enter` to execute the command.
+
+```shell
+$ hcdev init your_app_name
+```
+
+`hcdev` specifies that you wish to use the Holochain command line tools. `init` specifies to use the command for initializing a new project folder. `your_app_name` is an argument you supply as the app, and folder name.
```

Comment:
Basically all good and fine. I just want to express that this first sentence "hcdev specifies that [..] you use Holochain", and similarly the `$` above, feels like targeted to somebody who does not know how to use a shell.. which most likely would not be an app developer. So as a developer it feels a bit like not meant for me.. ;)

Maybe it's just me and I don't want to criticize your writing style which in general I enjoy very much. So this is not a strong opinion, just a soft expression of what I feel when reading this. Just to provide this as data :)